programs/steward/src/state/steward_state.rs

```diff
@@ -115,11 +123,24 @@ pub struct StewardState {
     /// Tracks whether unstake and delegate steps have completed
     pub rebalance_completed: U8Bool,
 
+    pub _padding0: [u8; STATE_PADDING_0_SIZE],
+
+    /// Number of validators added to the pool in the current cycle
+    pub validators_added: u16,
+
+    // TODO add closer to the top
+    /// Marks a validator for removal after `remove_validator_from_pool` has been called on the stake pool
+    /// This is cleaned up in the next epoch
+    pub validators_to_remove: BitMask,
```

Comment:
Best to move this up now imo since we'll need to do a state reset anyway, the validators_added could go before or after the U8Bools

Reply:
Done